docs/how-to-use-minio-server-side-encryption-with-aws-cli.md

```diff
@@ -1,88 +1,94 @@
+  --endpoint-url https://localhost:9000 \
+  --bucket my-bucket --key my-secret-diary \
+  --sse-customer-algorithm AES256 \
+  --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
+  --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
+  --body ~/my-diary.txt You should use your own encryption key.
+```
+
+In this example, a local Minio server is running on https://localhost:9000 with a self-signed certificate. TLS certificate verification is skipped using: `--no-verify-ssl`. If a Minio server uses a CA-signed certificate, then `--no-verify-ssl` should not be included, otherwise aws-cli would accept any certificate.
+
+
+## Display Object Information
+Specify the correct SSE-C key of an encrypted object to display its metadata:
+
+```sh
+Copy  aws s3api head-object \
   --no-verify-ssl \
   --endpoint-url https://localhost:9000 \
   --bucket my-bucket \
   --key my-secret-diary \
   --sse-customer-algorithm AES256 \
   --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
   --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg==
-  ```
-
-### 2.3 Download an object
-
-1. Now delete your local copy of `my-diary.txt`:  
-   `rm ~/my-diary.txt` 
-   
-2. You can restore the diary by downloading it from the server:
-   ```
-   aws s3api get-object \
-   --no-verify-ssl \
-   --endpoint-url https://localhost:9000 \
-   --bucket my-bucket \
-   --key my-secret-diary \
-   --sse-customer-algorithm AES256 \
-   --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
-   --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
-   ~/my-diary.txt
-   ```
-
-   
+```
+
+## Download an Object
+The following examples show how a local copy of a file can be removed and then restored by downloading it from the server:
+
+Delete your local copy of `my-diary.txt`:
+
+```sh
+rm ~/my-diary.txt
+```
+
+Restore the file by downloading it from the server:
+
+```sh
+Copyaws s3api get-object \
+--no-verify-ssl \
+--endpoint-url https://localhost:9000 \
+--bucket my-bucket \
+--key my-secret-diary \
+--sse-customer-algorithm AES256 \
+--sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
+--sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
+~/my-diary.txt
+```
+
+# <a name="security-notice"></a> Security-Related Notes
+
+* The Minio server will reject any SSE-C request made over an insecure (non-TLS) connection as per the S3 specification the. This means that SSE-C requires TLS / HTTPS, and an SSE-C request contains the encryption key. 
+* If an SSE-C request is made over a non-TLS connection, the SSE-C encryption key must be treated as compromised.
+* Per the S3 specification, the `content-md5` returned by an SSE-C PUT operation does not match the MD5 sum of the uploaded object. 
+* The Minio server uses a tamper-proof encryption scheme to encrypt objects and does not save the encryption key, which means you are responsible for managing encryption keys. If you lose the encryption key for an object, you will lose that object.
```

Comment:
"...you will lose the ability to decrypt that object"

Reply:
@eco-minio Revised as suggested.